docs/input/docs/usage/msbuild.md

```diff
@@ -50,6 +50,14 @@ The next thing you need to do is to remove the `Assembly*Version` attributes fro
 your `Properties\AssemblyInfo.cs` files. This puts GitVersion.MsBuild in charge of
 versioning your assemblies.
 
+For SDK-style projects, add the following properties set to `false`:
```

Comment:
Can you please explain why these properties need to be set to `false`?

Reply:
![img001](https://media.discordapp.net/attachments/682885060256727047/871773046573252628/Screenshot_20210801-1208262.png)
From https://elanderson.net/2017/06/duplicate-system-reflection-assemblycompanyattribute-attribute/

I has actually saw this screenshot I saved on my phone a few minutes and remembered to add it here.

Reply:
Thanks, but sorry that I wasn't clear: I was hoping you could describe it in the documentation (i.e. in this PR) and not just to me directly. :)

Reply:
Found some time to work on this again.
Sadly, there's nothing in the Microsoft Documentation that specifies the need to disable these property controls in favor of an alternative AssemblyInfo generator.
Aside from using an [Archive.org link](https://web.archive.org/web/20200918071836/https://elanderson.net/2017/06/duplicate-system-reflection-assemblycompanyattribute-attribute/), would you recommend any specific way of referring to Eric L. Anderson's post?

Reply:
We can just refer to [Microsoft's documentation](https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#generateassemblyconfigurationattribute), perhaps?

Reply:
Sure. Completed in https://github.com/BinToss/GitVersion/commit/8b969a3be6b523ea93f7ff0700eb6305a2a2371a